rich/__init__.py

```diff
@@ -1,7 +1,7 @@
 """Rich text and beautiful formatting in the terminal."""
 
 import os
-from typing import Callable, IO, TYPE_CHECKING, Any, Optional, Union
+from typing import IO, TYPE_CHECKING, Any, Callable, Optional, Union
```

Comment:
Not a change I would have introduced myself, but since I had to update this file pre-commit triggered the `isort` check on it, which spotted that slight imports ordering issue :slightly_smiling_face: